src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/AuthCheckerConfiguration.java

```diff
@@ -52,7 +52,7 @@ public void setAuthorisedServices(List<String> authorisedServices) {
 
     @Bean
     public Function<HttpServletRequest, Collection<String>> authorizedServicesExtractor() {
-        log.info(String.format("Configured authorised services: %s", String.join(", ", authorisedServices)));
+        //log.info(String.format("Configured authorised services: %s", String.join(", ", authorisedServices)));
```

Comment:
This was commented out instead of being removed as it may be of use for developers while working/debugging